css/button.css

```diff
@@ -0,0 +1,18 @@
+.button-main{
+    padding: 5px 10px 5px 10px;
+    border-radius: 10px;    
+    margin-left: 1rem;
+    margin-right: 1rem;
```

Comment:
Please use shorthand property for margin.

Reply:
Updated the values